lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureFileSystemConfig.java

```diff
@@ -47,6 +48,19 @@ public AzureFileSystemConfig setAuthType(AuthType authType)
         return this;
     }
 
+    @NotEmpty
+    public String getEndpoint()
+    {
+        return endpoint;
+    }
+
+    @Config("azure.endpoint")
```

Comment:
We should document this, including the default value and alternative example value for govcloud.